content/handbook/marketing/sales-plays/gitops/_index.md

```
@@ -0,0 +1,334 @@
+
+#### 評価中
+
+1. GitOps 関連の[顧客リファレンス](#customer-stories)を共有
+1. [GitOps の Pathfactory トラック](https://learn.gitlab.com/l/gitops-gtm-content)の一部であるテクニカルデモやウェビナーを共有
+1. [Gartner Peer Insights](/handbook/marketing/brand-and-product-marketing/product-and-solution-marketing/usecase-gtm/gitops/#gartner-peer-insights) を共有
+1. SA と協力して GitOps のテクニカルデモを披露
+1. 通話後、GitOps の会話のためにカスタマイズされた[GitOps デモメールテンプレート](https://docs.google.com/document/d/1uvZPw39OAExz7sIivIlPi3jNlUXKodVNy0mmIb-X0Cs/edit#)を顧客にメールで送信することを検討してください。
+
+#### 交渉 / 意思決定中
+
+1. [私たちがどう違うかを示す](#differentiators-how-gitlab-does-it-better)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)